dotcom-rendering/src/web/components/ArticleContainer.tsx

```diff
@@ -79,6 +79,26 @@ const articleAdStyles = css`
 	.ad-slot--fluid {
 		width: 100%;
 	}
+	.ad-slot--im {
```

Comment:
Should we keep a record of where these styles came from in frontend? None of the other ad slot styles defined here do

Reply:
I don’t think it’s essential. What matters most is that it works in DCR, doesn’t it?

For posterity, [here’s the search for `.ad-slot--im`](https://github.com/guardian/frontend/search?q=.ad-slot--im).

Reply:
I agree, it would be too much to mark every style that was used on both.